pom.xml

```diff
@@ -83,7 +83,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>github-branch-source</artifactId>
-            <version>2.3.6</version>
+            <version>2.5.4-rc860.9c636841daa2</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
why is this needed?

Reply:
A transitive dependency required when bumping the version for `github-branch-source` from `2.3.6` to `2.5.4...`. I can be more accurate so I'll copy and paste the enforce rules I got when bumping those versions later on.

Reply:
Why isn't it sucked in transitively by `github-branch-source`?

Reply:
bump @v1v , sorry I've not been more responsbile on this project, new kid, new house etc..

just bumping to see if you want to bump to the latest stable release of the github-branch-source dependency?